src/plugins/intel_cpu/src/nodes/interpolate.cpp

```diff
@@ -1786,35 +1787,46 @@ void Interpolate::initSupportedPrimitiveDescriptors() {
         supportedPrimitiveDescriptors.push_back({config, implDetail});
     };
 
-    const auto &dataMinDims = getInputShapeAtPort(DATA_ID).getMinDims();
-    bool isBlkApplied = getInputShapeAtPort(DATA_ID).getRank() > 1 && dataMinDims[1] != Shape::UNDEFINED_DIM && dataMinDims[1] > 1;
-
-    impl_desc_type impl_type;
+    impl_desc_type implType;
     if (mayiuse(cpu::x64::avx512_core)) {
-        impl_type = impl_desc_type::jit_avx512;
+        implType = impl_desc_type::jit_avx512;
     } else if (mayiuse(cpu::x64::avx2)) {
-        impl_type = impl_desc_type::jit_avx2;
+        implType = impl_desc_type::jit_avx2;
     } else if (mayiuse(cpu::x64::sse41)) {
-        impl_type = impl_desc_type::jit_sse42;
+        implType = impl_desc_type::jit_sse42;
     } else {
-        impl_type = impl_desc_type::ref;
+        implType = impl_desc_type::ref;
     }
 
-    if (!mayiuse(cpu::x64::sse41) || interpAttrs.mode == InterpolateMode::linear) {
-        pushDesc(LayoutType::ncsp, impl_type);
+    // planar for 1.ref on machine without sse41(if no sse41, canFuse() is false). 2.JIT kernel for f32 && avx2(gather).(with fuse)
```

Comment:
could you please update code explanation comments to cover planar support by emulation case.

Reply:
Done